packages/dashboard-core-plugins/src/panels/ColumnSelectionValidator.ts

```diff
@@ -0,0 +1,7 @@
+import { PanelComponent } from '@deephaven/dashboard';
+import { LinkColumn } from '../linker/LinkerUtils';
+
+export type ColumnSelectionValidator = (
```

Comment:
This file should probably be in the `linker` folder... or just even defined in `LinkerUtils`.